modules/flowable-ui/flowable-ui-idm-logic/src/main/java/org/flowable/ui/idm/service/keycloak/KeycloakConfiguration.java

```diff
@@ -55,7 +59,11 @@
     @Override
     public void afterPropertiesSet() throws Exception {
         Assert.hasText(server, "server must be set");
-        Assert.hasText(authenticationPassword, "authenticationPassword must be set");
+        if (isClientCredentials()){
```

Comment:
small nitpick: add space before the brace `{`

Reply:
Shall I fix in a new commit or ammed the old one?

Reply:
you can just fix it in a new commit to this PR

Reply:
Done